cluster/config-defaults.yaml

```diff
@@ -505,7 +505,7 @@ kubernetes_scheduler_image: "zalando"
 # when set to true, service account tokens can be used from outside the cluster
 allow_external_service_accounts: "false"
 # issue service account tokens with expiration time.
-rotate_service_account_tokens: "false"
+rotate_service_account_tokens: "true"
```

Comment:
Should we maybe enable it just in a couple of clusters first?

Reply:
Yes, I'll do that. The PR was to trigger the latest e2e tests with the other changes in here. I didn't realise how long it was ago since they last run. If it's green I'll enable it in one cluster and let it be for a while.